rules/python3.mk

```diff
@@ -0,0 +1,38 @@
+# python3.5.2+
```

Comment:
I agree with shuotian, should remove 3.5.2+. we specify 3.6.0-1 that what we have validated. For example, we might use something in this package that is only available for 3.6.0-1, then 3.5.2 is no longer supported. But the questions is since we have no plan to validate/test 3.5.2 in the future, when are we gong to know it will 3.5.2.